pgdb/v1/fts.go

```diff
@@ -108,6 +106,53 @@ func FullTextSearchVectors(docs []*SearchContent, additionalFilters ...jargon.Fi
 			_ = err
 		}
 	}
+	return rv
+}
+
+func camelSplitDocs(docs []*SearchContent) []lexeme {
+	rv := make([]lexeme, 0, 8)
+	camelCaseRegex := regexp.MustCompile(`[[:upper:]][[:lower:]]+`)
+	// acronymRegex - this is to catch acronyms in a named group 'token'
+	// the suffix after the group is foward looking to avoid the first letter of a camel cased word
+	// for example given the text AuthSNSTest this regex captures SNS in token but matches on SNSTe:
+	//
+	// Given the text `AuthSNSTest`:
+	// 		?P<token>[[:upper:]]+) -> SNS - captured in "token"
+	//		[[:upper:]][[:lower:]] -> Te - discarded
+	// Given the text `AuthSNS Foo`:
+	// 		?P<token>[[:upper:]]+) -> SNS - captured in "token"
+	//		\s -> the trailing empty space ' ' - discarded
+	// Given the text `AuthSNS`:
+	// 		?P<token>[[:upper:]]+) -> SNS - captured in "token"
+	//		$ -> end of string
+	acronymRegex := regexp.MustCompile(`(?P<token>[[:upper:]]+)([[:upper:]][[:lower:]]|$|\s)`)
+	template := "$token"
```

Comment:
module scoped const?

Reply:
Thank you, fixed in 4d22e33be3f06e9da0ead3c523e26131d1ca2695